### ISSUE TITLE
gdrive-linux-rpi caps speed at 2MBps (16Mbps)

### ISSUE THREAD

**Issue by mccbala** [prasmussen/gdrive#250]
While I am able to download at 50Mbps through my browser and close to 65Mbps through desktop (ubuntu - gdrive-linux-x64), the download rate stays at 16Mbps in gdrive-linux-rpi. I have tried with multiple connections.  Is this an intentional limitation?

**Comment by chuanzgh** [comment]
I don't see the purpose and benefits of intentionally limiting the speed for a particular platform? If you're using the vanilla version of the pi (i.e. you did not do any mods to the networking on the pi), it's more likely that you've hit some sort of hardware speed cap on the pi itself.

**Comment by mccbala** [comment]
True. I assumed it might've been done in consideration to the hardware
capabilities of Pi. All other downloads happen in full speed through wget &
curl.

I'm running the latest Raspbian Jessie Lite on RPi2. Ethernet connection.

Is there a way I can find out some debug info on this? A verbose mode or a
log file that I could post here?

**Comment by mccbala** [comment]
Sadly no.
